client/src/App.css

```diff
@@ -1,4 +1,4 @@
-#root {
+/* #root {
```

Comment:
Unless you're keeping this code for a reason, I would delete unnecessary commented out code.